src/Orleans/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Newtonsoft.Json": "7.0.1",
+    "Newtonsoft.Json": "9.0.1",
```

Comment:
Yeah I knew you will caught this ! Just prompt up to ask why Olreans use a obsolete version while test project use a newer one and test project picked up the old version in runtime. I guess for now  I will make test project to use the new version then. 

when we normally update core framwork's dependency version?